src/pages/taskHistory/BuildVariantSelector.tsx

```diff
@@ -71,6 +71,7 @@ export const BuildVariantSelector: React.FC<BuildVariantSelectorProps> = ({
         searchFunc={handleSearch}
         optionRenderer={(option: BuildVariantTuple, onClick, isChecked) => (
           <SearchableDropdownOption
+            key={`searchable_dropdown_option_${option.buildVariant}`}
```

Comment:
Added to fix this problem:

<img width="396" alt="Screen Shot 2022-01-25 at 12 23 51 PM" src="https://user-images.githubusercontent.com/47064971/151035785-e8932afc-6db2-4932-b266-e408dc325fde.png">